operator/src/test/resources/runDeployedTests.sh

```diff
@@ -0,0 +1,15 @@
+cd ../../..
+eval $(minikube docker-env)
+kubectl create namespace keycloak-test
+
+mvn clean package -Doperator \
+    -Dquarkus.container-image.build=true \
+    -Dquarkus.kubernetes.deploy=true \
+    -Dquarkus.kubernetes.namespace=keycloak-test \
+    -Dquarkus.kubernetes-client.namespace=keycloak-test \
+    -Dquarkus.kubernetes.deployment-target=kubernetes,openshift,minikube \
+    -DskipTests
+mvn test -Dtest=OperatorE2ERemoteIT
```

Comment:
Does `OperatorE2ERemoteIT` runs even when, eventually, running unit tests?
Should we exclude `IT` tests from "default" tests?

Reply:
nop..... surefire doesnt run files without Test at the end or beginning

Reply:
perfect thanks for the explanation 👍